# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/CanvasRenderer.h
# Commit: ae8e94c8f819
# Full Hash: ae8e94c8f819bd46c31b6fbee1f2cdb5580c9e1f
# Author: Jamie Nicol <jnicol@mozilla.com>
# Date: 2024-07-23 09:34:47
# Regressor Bug: 1898238
# File Overlap Count: 1
# Description:
#   Bug 1898238 - Use SharedSurface_EGLImage for out-of-process webgl and accelerated canvas on Android. r=sotaro,jgilbert
#   
#   Adds a pref wegb.enable-egl-image which is true by default. When
#   enabled, we use EGL images as the preferred texture type for
#   out-of-process webgl and accelerated canvas on Android. In-process
# ==============================================================================

diff -r 549abd685d11 -r ae8e94c8f819 gfx/layers/CanvasRenderer.h
--- a/gfx/layers/CanvasRenderer.h	Mon Jul 22 16:02:34 2024 +0000
+++ b/gfx/layers/CanvasRenderer.h	Mon Jul 22 16:02:34 2024 +0000
@@ -30,7 +30,7 @@
 class PersistentBufferProvider;
 class WebRenderCanvasRendererAsync;
 
-TextureType TexTypeForWebgl(KnowsCompositor*);
+TextureType TexTypeForWebgl(KnowsCompositor*, bool aIsWebglOop);
 
 struct CanvasRendererData final {
   CanvasRendererData();